### PR TITLE
fix: Token renewal button [issue #3033]

### PR DESCRIPTION
**Pull request by Ben-Epstein**
fixes https://github.com/heartexlabs/label-studio/issues/3033

### CODE REVIEW

File: label_studio/users/templates/users/user_account.html [hunk 1]
```diff
@@ -176,9 +176,9 @@
   <script>
     (() => {
       {% if settings.HOSTNAME %}
```
Comment (wesleylima) [inline label_studio/users/templates/users/user_account.html:178]: Hi @Ben-Epstein. Thanks for the fix!

What do you think think of adding another javascript const ?

```suggestion
      {% if settings.HOSTNAME %}
      const hostname = {{ settings.HOSTNAME }};
      {% else %}
      const hostname = 'http://localhost:8080';
      {% endif %}
```

Comment (Ben-Epstein) [reply label_studio/users/templates/users/user_account.html:178]: hey, sure! Works for me, and i think this is cleaner.

File: label_studio/users/templates/users/user_account.html [hunk 2]
```diff
@@ -176,9 +176,13 @@
...
-      const CURL_TEMPLATE = "curl -X GET {{ settings.HOSTNAME }}/api/projects/ -H 'Authorization: Token'"
+      const hostname = {{ settings.HOSTNAME }};
       {% else %}
-      const CURL_TEMPLATE = "curl -X GET http://localhost:8080/api/projects/ -H 'Authorization: Token'"
+      const hostname = 'http://localhost:8080';
+      {% endif %}
+      const CURL_TEMPLATE = "curl -X GET {{ settings.HOSTNAME }}/api/projects/ -H 'Authorization: Token"
+      {% else %}
+      const CURL_TEMPLATE = "curl -X GET http://localhost:8080/api/projects/ -H 'Authorization: Token"
```
Comment (wesleylima) [inline label_studio/users/templates/users/user_account.html:185]: `CURL_TEMPLATE` isn't necessary anymore. These lines can be removed.

Comment (Ben-Epstein) [reply label_studio/users/templates/users/user_account.html:185]: why is the curl template not necessary?

Comment (wesleylima) [reply label_studio/users/templates/users/user_account.html:185]: `CURL_TEMPLATE` was being used to set `example.value` on line 205 before this change. Now example.value is being constructed using the `hostname` const.

**Comment by makseq** [comment]
Please rename this PR to more clear, "fix 3033" is saying nothing..

**Comment by Ben-Epstein** [comment]
@makseq i updated it, but the explanation of the issue is in the issue, which is linked.

File: label_studio/users/templates/users/user_account.html (hunk 2 above)
Comment (Ben-Epstein) [reply label_studio/users/templates/users/user_account.html]: updated @wesleylima

**Review by wesleylima — APPROVED** [review]

**Pull request merged**